CMP1903_A1_2324/Program.cs

```diff
@@ -14,6 +15,28 @@ static void Main(string[] args)
              * Create a Game object and call its methods.
              * Create a Testing object to verify the output and operation of the other classes.
              */
+
+            /// <summary> Main method. </summary>
+            /// <remarks> Creates a Game object and calls its methods. Creates a Testing object to verify the output and operation of the other classes. </remarks>
+            /// <param name="game"> The game object. </param>
+            /// <param name="test"> The testing object. </param>
+            /// <param name="x"> The amount of dice to roll. </param>
+            /// <param name="game"> The game object. </param>
+
+            Game game = new Game();
+            Testing test = new Testing();
+
+            Console.WriteLine("How many dice would you like to roll..?");
+            int x = Convert.ToInt32(Console.ReadLine());
```

Comment:
The game is required to roll all 3 dice. To meet the requirement, you should not let the user to choose the amount.

You should implement code to handle exceptions and erroneous inputs. (While testing your program, I found that it was crashed when I input a letter, and it generated and rolled more than 3 dice when I input an integer which is greater than 3).

Try to name the variable with some representative name (try to avoid 'x', 'y',...).